test/Feature/DeterministicSwitch.c

```diff
@@ -0,0 +1,41 @@
+// The order cases are generated in LLVM 2.9 is different: tab first then space
```

Comment:
If the ordering is a problem we could have a separate version of the test for LLVM 2.9. However seeing as I would like LLVM 2.9 deprecated ASAP I'm fine with how this is right now.